admin/app/controllers/solidus_admin/auth.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module SolidusAdmin::Auth
+  extend ActiveSupport::Concern
+
+  # Support the legacy Solidus Backend authentication method
+  include SolidusAdmin::BackendAuthSupport
```

Comment:
Given that `SolidusAdmin::BaseController` includes `SolidusAdmin::Auth`, we include `BackendAuthSupport` no matter what users configure. I know this is in the preliminary phase, but have you thought about how we would deal with this `include` line to be dependent on the configuration? I'm thinking that maybe what should be configurable is the name of the module, and that we should expect it to define the expected interface. Then, for now, we can default to `SolidusAdmin::BackendAuthSupport`. Does it make sense?

```ruby
# admin/lib/solidus_admin/configuration.rb

class_name_attribute  :authentication_module, default: `SolidusAdmin::BackendAuthSupport`
```